README.md

```diff
@@ -363,6 +363,12 @@ table {
 ```
 
 This option is off by default.
+=======
```

Comment:
Did you mean to add these equal signs here?

Reply:
Sorry that was a leftover from a merge conflict, now removed!

(there are linting errors too that I can't reproduce on my local machine... I'll try to clean up this PR over the next day)